integration

```diff
@@ -148,6 +148,7 @@
   "chises/ha-oilfox",
   "Chouffy/home_assistant_libratone_zipp",
   "Chouffy/home_assistant_tgtg",
+  "chris-mc1/qBittorrent-hass",
```

Comment:
<https://github.com/chris-mc1/qBittorrent-hass/blob/main/README.md>
There is no instructions on how to configure this integration.

Reply:
Update Readme, added instructions on Setup